packages/api/src/external/fhir/document/index.ts

```diff
@@ -144,3 +156,110 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
     use: id.use === "unspecified" ? undefined : id.use,
   };
 }
+
+/**
+ * Takes a CW resource and converts it to a FHIR resource
+ *
+ * @param resource
+ * @returns
+ */
+function convertToFHIRResource(resource: Contained): Resource | undefined {
+  if (resource.resourceType === ResourceType.Patient && resource.id) {
+    return {
+      resourceType: ResourceType.Patient,
+      id: resource.id,
+    };
```

Comment:
We already add the Patient when we declare the `containedContent` constant on line 75, right? Wondering if we can have additional Patients... if that's the case, then we should accept Patients w/o `id`, and include the ref and other properties.

Reply:
Oh, you're right. In my head, it doesn't make sense that one document will be referring to more than 1 patient. This would be kinda invading the other patient's privacy and I'm just not aware of any group tests like that 😄 But perhaps I'm just missing some use cases?